### DISCUSSION TITLE
mindsdb api create datasource question

### DISCUSSION THREAD

**Discussion by henry145145** [mindsdb/mindsdb#1967] [Get help] [answered]
Hi,
I would like to know is there an example of an api call that can reproduce the steps in here https://docs.mindsdb.com/datasources/snowflake/

I have been looking at this https://apidocs.mindsdb.com/?version=latest#5cb802e0-d458-4ffc-9fee-fbf77b69baa6 but I am not sure which arguments I need to pass into the body in order to achieve the example above. For example, how would I pass in "Query"?

Also, is there also an example for creating datasource from sdk? I have been reading this https://github.com/mindsdb/mindsdb_js_sdk#readme but not find so far.

Thank you for reading.

**Comment by ZoranPandovski** [comment, marked as answer]
Hey @henry145145 that page in our docs was removed, but looks like it is still indexed on search. With the latest version of MindsDB, the data sources are just connection to the database where you pass the SELECT statement to get the data, you can check the code https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/api/httphttps://github.com/mindsdb/mindsdb/tree/staging/mindsdb/api/http. Is it there specific requirements for using the REST API? If not, I would suggest you use our SQL API because it is well supported and maintained.
